boards/arm/stm32l4/nucleo-l432kc/src/stm32_dac.c

```diff
@@ -0,0 +1,97 @@
+/*****************************************************************************
+ * boards/arm/stm32l4/nucleo-l432kc/src/stm32_dac.c
+ *
+ *   Copyright (C) 2017 Haltian Ltd. All rights reserved.
```

Comment:
@daniel-p-carvalho could you create a new patch to change the copright to Apache?

Reply:
@xiaoxiang781216 this file was copied from nucleo-l452re so I think that I don't have the rights to do the copyright change unless Haltian Ltd. give the permission.

Reply:
@juniskane did you signed the ICLA or Haltian signed the SGA?

Reply:
I think that this change must be reverted.  We cannot being ANY new files into the repositories that do not have Apache 2.0 license.

UNLESS we follow the rules for third party software.  If it is not Apache 2.0, it is third party and not truly a part of NuttX.